renku/core/management/workflow/plan_factory.py

```diff
@@ -125,15 +126,15 @@ def split_command_and_args(self):
             # only guess subcommand for more arguments
             return cmd, args
 
-        while args and re.match(self._RE_SUBCOMMAND, args[0]) and not self.is_existing_path(args[0]):
+        while args and re.match(self._RE_SUBCOMMAND, args[0]) and not self.is_existing_path(args[0])[0]:
             cmd.append(args.pop(0))
 
         return cmd, args
 
     def is_existing_path(self, candidate, ignore=None):
         """Return a path instance if it exists in current directory."""
         if ignore and candidate in ignore:
-            return
+            return None, False
```

Comment:
Should we create a new functions that just does this check since most of the time we only use the first returned value?

Reply:
I had that at one point, but then it meant doing the check twice, once in `is_existing_path` and once in `is_relative_to_client` or whatever it'd be called. 

Come to think of it, right now the `is_existing_path` does more than the name implies. Probably should be properly refactored a bit to make it more clear what it actually does. `"""Return a path instance if it exists in current directory."""` is definitely not what this does 🙂 It's more `resolve_path()` or something like that if I read it correctly.

Feel free to change it if you want

Reply:
Will do!